drivers/driver.go

```diff
@@ -53,3 +47,15 @@ func Register(driverName string, driver Driver) {
 	defer driversMu.Unlock()
 	registeredDrivers[driverName] = driver
 }
+
+func GetDriver(driverName string) (Driver, error) {
+	driversMu.Lock()
+	defer driversMu.Unlock()
+
+	driver, ok := registeredDrivers[driverName]
+	if !ok {
+		return nil, fmt.Errorf("driver %s not found", driverName)
```

Comment:
nit to have the driver name surrounded by quotes for easy reading

```suggestion
		return nil, fmt.Errorf("driver %q not found", driverName)
```